vizro-core/examples/visual-vocabulary/custom_charts.py

```diff
@@ -193,3 +193,43 @@ def radar(data_frame, **kwargs) -> go.Figure:
     fig = px.line_polar(data_frame, **kwargs)
     fig.update_traces(fill="toself")
     return fig
+
+
+@capture("graph")
+def dumbbell(data_frame: pd.DataFrame, x: str, y: str, color: str) -> go.Figure:
+    """Creates a dumbbell chart using Plotly's `px.scatter` and `add_shape`.
+
+    A dumbbell plot is a type of dot plot where the points, displaying different groups, are connected with a straight
+    line. They are ideal for illustrating differences or gaps between two points.
+
+    Args:
+        data_frame (pd.DataFrame): The data source for the chart.
+        x (str): Column name in `data_frame` for x-axis values.
+        y (str): Column name in `data_frame` for y-axis values.
+        color (str): Column name in `data_frame` used for coloring the markers.
+
+    Returns:
+        go.Figure: A Plotly Figure object representing the dumbbell chart.
+
+    Inspired by: https://community.plotly.com/t/how-to-make-dumbbell-plots-in-plotly-python/47762
+
+    """
+    # Add two dots to plot
+    fig = px.scatter(data_frame, y=y, x=x, color=color)
+
+    # Add lines between dots
+    for i in data_frame[y].unique():
+        df_sub = data_frame[data_frame[y] == i]
+        fig.add_shape(
+            type="line",
+            layer="below",
+            y0=df_sub[y].values[0],
+            x0=df_sub[x].values[0],
+            y1=df_sub[y].values[1],
+            x1=df_sub[x].values[1],
+            line_color="grey",
+        )
+
+    # Increase size of dots
+    fig.update_traces(marker_size=12)
```

Comment:
The line currently feels a bit thin to me but not a strong feeling. I guess lollipop and this plot should have the same line thickness, whatever that is.

Reply:
I made it thicker 👍 The Lollipop is not merged, but i took the same numbers now